cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -201,7 +201,7 @@
     }, {
         'titles': ['Deputy General Counsel', 'Assistant Director'],
         'slug':   'litigation-and-oversight',
-        'name':   'Vacant',
+        'name':   '(continuing) Vacant',
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/6c523ee9eea789273e70555bea46bdb6657e4d8a